docs/index.md

```diff
@@ -1,8 +1,24 @@
 # TorchServe
 
-TorchServe is a performant, flexible and easy to use tool for serving PyTorch eager mode and torschripted models.
+TorchServe is a performant, flexible and easy to use tool for serving PyTorch eager mode and torchscripted models.
```

Comment:
I don't think torchscript should be in in our one liner anymore

Reply:
Should we mention compiled mode in replacement of torchscript?

Reply:
Thanks. I'm going to just modify it to "serving PyTorch models in production"